byterun/major_gc.c

```diff
@@ -337,7 +337,7 @@ static uintnat default_slice_budget() {
   */
   uintnat heap_size = caml_heap_size(Caml_state->shared_heap);
   heap_words = (double)Wsize_bsize(heap_size);
-  uintnat heap_blocks = caml_heap_blocks(Caml_state->shared_heap);
+  uintnat heap_sweep_size = heap_size;
```

Comment:
Should this be `heap_words`? `heap_size` is the size of the heap in bytes.

Reply:
Whoops! You are absolutely right, apologies.